cli/cli.py

```diff
@@ -21,10 +31,51 @@
 
 
 def list_resources(provider: str, resource_type: str):
+    print_banner(False)
     if resource_type == "services":
         print_services(list_services(provider))
     elif resource_type == "fixers":
         print_fixers(list_fixers(provider))
+    elif resource_type == "categories":
+        print_categories(list_categories(bulk_load_checks_metadata(provider)))
+    elif resource_type == "compliance":
+        print_compliance_frameworks(bulk_load_compliance_frameworks(provider))
+    elif resource_type in ["checks", "checks-json"]:
+        bulk_checks_metadata = bulk_load_checks_metadata(provider)
+        # TODO fill the rest of the arguments
+        checks_to_execute = load_checks_to_execute(
+            bulk_checks_metadata,
+            bulk_load_compliance_frameworks(provider),
+            None,
+            [],
+            [],
+            [],
+            [],
+            [],
+            provider,
+        )
+        if resource_type == "checks":
+            print_checks(provider, sorted(checks_to_execute), bulk_checks_metadata)
+        elif resource_type == "checks-json":
+            print(list_checks_json(provider, sorted(checks_to_execute)))
+
+
+def list_compliance_requirements(
+    provider: str, compliance_frameworks: list[str] = None
+):
+    print_banner(False)
```

Comment:
Since this is not being done in the current CLI let's remove it.
```suggestion
```

Reply:
Done